docs/sql-ref-syntax-aux-show-table.md

```diff
@@ -69,10 +69,10 @@ INSERT INTO employee PARTITION (grade = 2) VALUES ('suj');
 
  -- Show the details of the table
 SHOW TABLE EXTENDED LIKE 'employee';
-+--------+---------+-----------+--------------------------------------------------------------+
-|database|tableName|isTemporary|                         information                          |
```

Comment:
do this change is  necessary? i found that there are similar problem in other `show *` doc, if this is necessary, i will fix these in follow pr, or not, will close this title.